### DISCUSSION TITLE
Error when running timeseries tutorial

### DISCUSSION THREAD

**Discussion by osteinm** [e2nIEE/pandapipes#368] [Q&A] [answered]
While trying to run the timeseries example from the tutorial file (https://github.com/e2nIEE/pandapipes/blob/master/tutorials/simple_time_series_example.ipynb), I received 'KeyError: 0'. I am expecting this error to come from the line 'time_steps = range(10)'. When I change that line to range(1,10) the error message is  'KeyError: 1'. Have you experienced such an error before?

Pandapower and pandapipes are freshly upgraded.

**Comment by jkisse** [comment, marked as answer]
I just tried the current version of that [tutorial on binder](https://mybinder.org/v2/gh/e2nIEE/pandapipes/master?filepath=tutorials), and it is working fine. 
I think the error might be related to your profile data that is imported from csv files. It uses the default values for separator (comma) and decimal points. 
`profiles_sink = pd.read_csv(os.path.join('files',
                                         'simple_time_series_example_sink_profiles.csv'),
                                         index_col=0)`
Please make sure that your profile data frames (profiles_sink, profiles_source) have numeric indices, starting from 0.

**Reply by osteinm** [reply]
Ok, got it running - thanks!
